src/index.js

```diff
@@ -914,6 +914,18 @@ Collection.prototype.all = function () {
   return this.items;
 };
 
+Collection.prototype.concat = function (collection) {
+  if (collection instanceof Collection) {
+    collection = collection.all();
+  }
+
+  for (let iterator = 1; iterator <= collection.count(); iterator++) {
+    this.items.push(collection[iterator]);
```

Comment:
Needs logic for when item is of different type
- ``Array``
- ``Object``
- ``String``
- etc.